### ISSUE TITLE
App does not launch without --disable-gpu-sandbox

### ISSUE THREAD

**Issue by VortexAcherontic** [itchio/itch#2885]
Hello there just installed the Itch app and noticed after the setup completed the installed app did not launched.
Since Electron seems to be the go-to software for app launcher these days I just tried adding `--disable-gpu-sandbox` to the itch executable and it worked.

If I am not mistaken this is an issue with older version of electron on some distributions which fall into one or more of the following categories:
- Uses systemD 250 (251?) or newer
- Runs Wayland (can also happen on X11)
- Runs closed source nVidia drivers

Not 100% sure which combinations of the above is the true root cause but I believe this to be fixed up-stream in Electorn. But I am not 100% sure if this is really the case or I am just missing some information.
Neither way adding --disble-gpu-sandbox to the executable or to the *.desktop file makes it work.

At this point I believe this could be fixed by "just" updating to a newer version of Electron.
Itch seems to run on Electron 10.4.5 (according to the log below) and the [latest release is 21.3.1](https://releases.electronjs.org/releases/stable)

Log of crashed launch:

````
{"time":1669375850464,"level":30,"msg":"itch@25.5.1 on electron@10.4.5 in production"}
{"time":1669375850465,"level":20,"msg":"imported preferences: {\"downloadSelfUpdates\":true,\"offlineMode\":false,\"installLocations\":{},\"defaultInstallLocation\":\"d5f1ce95-5902-4165-8d41-0628bd402155\",\"isolateApps\":false,\"closeToTray\":true,\"readyNotification\":true,\"showAdvanced\":false,\"openAtLogin\":false,\"openAsHidden\":false,\"manualGameUpdates\":false,\"preventDisplaySleep\":true,\"preferOptimizedPatches\":false,\"disableBrowser\":false,\"enableTabs\":false,\"lastSuccessfulSetupVersion\":\"25.5.1\",\"importedOldInstallLocations\":true,\"gotMinimizeNotification\":true}","name":"m/reac/preb/load-preferences"}
{"time":1669375850568,"level":20,"msg":"imported preferences: {\"downloadSelfUpdates\":true,\"offlineMode\":false,\"installLocations\":{},\"defaultInstallLocation\":\"d5f1ce95-5902-4165-8d41-0628bd402155\",\"isolateApps\":false,\"closeToTray\":true,\"readyNotification\":true,\"showAdvanced\":false,\"openAtLogin\":false,\"openAsHidden\":false,\"manualGameUpdates\":false,\"preventDisplaySleep\":true,\"preferOptimizedPatches\":false,\"disableBrowser\":false,\"enableTabs\":false,\"lastSuccessfulSetupVersion\":\"25.5.1\",\"importedOldInstallLocations\":true,\"gotMinimizeNotification\":true}","name":"m/reac/preb/load-preferences"}
{"time":1669375850574,"level":20,"msg":"Registering itch: protocol for session with user agent Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) itch/25.5.1 Chrome/85.0.4183.121 Electron/10.4.5 Safari/537.36","name":"m/net/register-itch-protocol"}
{"time":1669375850609,"level":30,"msg":"preboot ran in 42.00ms","name":"m/reac/preboot"}
{"time":1669375850610,"level":20,"msg":"Updating login item settings, open=false, hidden=false","name":"m/reac/open-at-login"}
{"time":1669375850610,"level":20,"msg":"Copying /usr/share/applications/io.itch.itch.desktop => /home/vortexacherontic/.config/autostart/io.itch.itch.desktop","name":"m/reac/open-at-login"}
{"time":1669375850612,"level":20,"msg":"Waiting on root window to dispatch boot","name":"m/reac/winds"}
````

System information:

- OS: openSUSE MicroOS
- Kernel: 6.0.8
- Display server: Wayland 1.21.0
- Desktop Environment: Gnome 43.1
- nVidia Driver: 525.53
- systemD: 252.1

**Comment by lockie** [comment]
Affected by this as well on Gentoo. Any chance of fixing?

**Comment by VortexAcherontic** [comment]
I just re-tested this and using XWayland the client does work properly.
Only if forcing Electron to enable the Wayland backend see #2976 the app does crash and not even --disable-gpu-sandbox helps.

Did you run itch on XWayland or native Wayland?

**Comment by lockie** [comment]
@VortexAcherontic I'm not using wayland at all.

**Comment by leafo** [comment]
Closing out old tickets for very old versions of the app. Please update to 26.9.0 or above and create a new issue if you are still having problems.
